aodn_cloud_optimised/bin/anmn_hourly_timeseries.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+import importlib.resources
+
+from aodn_cloud_optimised.lib.CommonHandler import cloud_optimised_creation_loop
+from aodn_cloud_optimised.lib.config import load_variable_from_config, load_dataset_config
+from aodn_cloud_optimised.lib.s3Tools import s3_ls
+from aodn_cloud_optimised.lib.AnmnHourlyTsHandler import AnmnHourlyTsHandler
+
+if __name__ == "__main__":
+
+    BUCKET_RAW_DEFAULT = load_variable_from_config('BUCKET_RAW_DEFAULT')
+
+    nc_obj_ls = s3_ls(BUCKET_RAW_DEFAULT, 'IMOS/ANMN/NSW') + \
```

Comment:
@lbesnard Missing 'IMOS/ANMN/NRS' prefix here! (The National Referece Stations -- quite important IMOS data)